mitosheet/mitosheet/tests/step_performers/test_export_to_file.py

```diff
@@ -42,6 +42,51 @@
     "conditional_formats": []
 }
 
+CONDITIONAL_FORMATS = [
```

Comment:
Check out pytest.mark.parameterize -- and how we use it through our codebase in mitosheet/tests. I think it's going to be super helpful for scaling tests to multiple filter conditions - especially if we have the above mentioned utility.